config/initializers/menus.rb

```diff
@@ -72,6 +72,32 @@
             html: { accesskey: OpenProject::AccessKeys.key_for(:help),
                     title: I18n.t("label_help"),
                     target: "_blank" }
+
+  menu.push :home,
+            { controller: "/homescreen", action: "index" },
+            context: :my,
+            icon: "home",
+            first: true
+
+  menu.push :my_page,
+            { controller: "/my/page", action: "show" },
+            context: :my,
+            after: :home,
+            icon: "person",
+            caption: I18n.t("my_page.label"),
+            if: ->(*) do
+              User.current.logged?
+            end
+
+  menu.push :my_time_tracking,
+            { controller: "/my/time_tracking", action: "index" },
+            after: :my_page,
+            context: :my,
+            caption: :label_my_time_tracking,
+            if: ->(*) do
+              User.current.allowed_in_any_project?(:log_own_time) || User.current.allowed_in_any_project?(:log_time)
+            end,
+            icon: :stopwatch
```

Comment:
Please move that to the costs engine where the other `time_tracking` menu entries are listed.